apps/server/src/modules/oauth/service/oauth.service.spec.ts

```diff
@@ -16,7 +16,9 @@ import { Configuration } from '@hpi-schul-cloud/commons';
 import { schoolFactory } from '@shared/testing';
 import { DefaultEncryptionService, SymetricKeyEncryptionService } from '@shared/infra/encryption';
 import { AuthorizationParams } from '@src/modules/oauth/controller/dto/authorization.params';
-import { IservOAuthService } from './iserv-oauth.service';
+import { ProvisioningDto } from '@src/modules/provisioning/dto/provisioning.dto';
+import { UnprocessableEntityException } from '@nestjs/common';
```

Comment:
I have cleared them up. In one case i would throw a notfound and the other case would be better to throw a badrequest exception because of wrong data of the client